### ISSUE TITLE
RES has problem downloading images from posts starting with an emoticon 

### ISSUE THREAD

**Issue by ThePiachu** [honestbleeps/Reddit-Enhancement-Suite#5354]
- My browser (copy from https://www.whatismybrowser.com): Chrome 95 on Windows 10
- My RES version: v5.22.5
- Am I in reddit beta (yes/no): No

*What's the problem?*

Whenever I try using the save icon on images that start with an emoticon, RES doesn't save them. I'm guessing there is some issue specifically with file names that start with an emoticon, since I can save files that contain an emoticon just fine. For a good subreddit to try this out, check https://www.reddit.com/r/NatureIsFuckingLit/ :

![image](https://user-images.githubusercontent.com/964559/139370927-cbbd7c0d-46f9-4ea6-a2e1-cdbe45ff896f.png)

(cursor not visible) I click that icon and nothing is happening.

*Are there any related posts in https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues or /r/RESissues?*

No

*What other browser extensions are installed?*

- HTTPS Everywhere
- Privacy Badger
- uBlock Origins

**Comment by maryarankin** [comment]
Can I work on this issue?

**Comment by honestbleeps** [comment]
absolutely, we need more folks willing to contribute bug fixes, etc! thank you!
